src/test/java/org/ice4j/ice/harvest/SinglePortUdpHarvesterTest.java

```diff
@@ -113,4 +113,25 @@ public void testRebindWithClose() throws Exception
             }
         }
     }
+
+    @Test
+    public void testBindWithPortZero() throws Exception {
```

Comment:
nit: In java code we always put braces on a new line (also applies below).

Reply:
Thanks. I'll fix it in the next push.

Should we add automated linting to ice4j as we have for other projects as well to catch issues like this?